code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -649,7 +649,7 @@
 	to_chat(src, "<span class='notice'>You command [T], and they will obey.</span>")
 	emote("me", 1, "whispers.")
 
-	vampire.use_blood(25)
+	vampire.use_blood(150)
```

Comment:
You need to update the cost on line 605 aswell. Right now it would still be displayed as "Dominate (25)"